contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -1100,6 +1100,109 @@ AND sys.babelfish_get_pltsql_function_signature(ao.object_id) = f.funcsignature
 WHERE ao.type in ('P', 'RF', 'V', 'TR', 'FN', 'IF', 'TF', 'R');
 GRANT SELECT ON sys.all_sql_modules_internal TO PUBLIC;
 
+-- deprecate old FOR XML/JSON functions if they exist - if this is a fresh install of 3.1 they won't exist, so check for that
```

Comment:
The comment needs be updated, e.g., old functions only exist when upgrade paths do not contain v2.4 or higher.

Reply:
updated